document/src/main/java/com/yahoo/document/DocumentId.java

```diff
@@ -33,6 +33,11 @@ public DocumentId(String id) {
         if (id == null) {
             throw new IllegalArgumentException("Cannot create DocumentId from null id.");
         }
+        if (id.length() > IdString.MAX_LENGTH) {
+            throw new IllegalArgumentException("The document id(" + id.length() + ") is too long(65536). " +
```

Comment:
Should the reported max length be `IdString.MAX_LENGTH` instead of hardcoded in the string?

Reply:
Correct, fixed